package.json

```diff
@@ -12,8 +12,9 @@
     "typescript": "^3.7.2"
   },
   "scripts": {
-    "build": "yarn check-deps && yarn build:interfaces && polkadot-dev-build-ts",
+    "build": "yarn check-deps && yarn build:interfaces && polkadot-dev-build-ts && yarn build:metadata",
```

Comment:
`build:metadata` requires a running node, which won't exists in CI

Reply:
Will remove it.